### ISSUE TITLE
Strange Boolean behaviour

### ISSUE THREAD

**Issue by ktimothy** [solnic/virtus#377]
```
irb(main):016:0> Virtus::Attribute.build(true).is_a?(Virtus::Attribute::Boolean)
=> true
irb(main):017:0> Virtus::Attribute.build(false).is_a?(Virtus::Attribute::Boolean)
=> false
```
Found it out while investigating a bug in grape.

**Comment by acook** [comment]
Yeah, looks like FalseClass isn't listed as a primitive for it: https://github.com/solnic/virtus/blob/master/lib/virtus/attribute/boolean.rb#L18

**Comment by ktimothy** [comment]
It also looks like only one primitive type can be set.

_(Oh, Ruby, TrueClass and FalseClass, seriously? =))_

**Comment by envygeeks** [comment]
Ruby has no primitive types.

**Comment by ktimothy** [comment]
I meant `primitive` attribute of `Virtus::Attribute` class.

**Comment by acook** [comment]
@envygeeks `Virtus::Attribute` has a `primitive` accessor: https://github.com/solnic/virtus/blob/master/lib/virtus/attribute.rb#L51
